spec/factories/offenders.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :offender, class: 'Nomis::Offender' do
+    inprisonment_status do 'SENT03' end
+    sentence do
+      Nomis::SentenceDetail.new.tap { |s|
+        s.automatic_release_date = Time.zone.today + 1.year
+        s.sentence_start_date = Time.zone.today
+      }
+    end
+    prison_id do 'LEI' end
+    case_allocation do 'NPS' end
+    mappa_level { 0 }
```

Comment:
Could these block declarations be made consistent? I think the preferred approach is `{...}` curly braces for single line blocks, and `do...end` for multiline.

```suggestion
    prison_id { 'LEI' }
    case_allocation { 'NPS' }
    mappa_level { 0 }
```

Reply:
I think rubocop -a is doing this - agree with your suggestion above